tests/src/core/resolve.js

```diff
@@ -113,7 +113,7 @@ describe('resolve', function () {
   });
 
   it('finds resolvers from the source files rather than eslint-module-utils', function () {
-    const testContext = utils.testContext({ 'import/resolver': { foo: {} } });
+    const testContext = utils.testContext({ 'import/resolver': { './foo-bar-resolver-v2': {} } });
```

Comment:
why these changes?

Reply:
It looks like the test was incorrectly setup. At one point during these changes, these tests were failing.

Reply:
they pass currently tho (just not on windows), so i'd expect them to continue to pass

Reply:
I reverted the changes. I don't remember what configuration I was in, but these tests were failing and when I looked at the testContext code, it made sense to me why it was failing. The resulting require module name was something like `node_modules/eslint-import-utils-foo` and both the test description and assertion expect the path to be from the "source files". So I changed it and it passed. But now that I run the tests, I don't see a failure error in the logs.